EnglishTranslation-master/SKILL.tsv

```diff
@@ -6209,44 +6209,44 @@ SKILL_20160224_006208	{#993399}{ol}[Magic]{/}{/}All Ice-type magic cast by you a
 SKILL_20160224_006214	Resistance Effect Chance: +#{CaptionRatio}#%{nl}Duration: #{CaptionTime}# seconds{nl}Consumes 1 Rune Stone{nl}Cast Time: 8 seconds
-SKILL_20160224_006215	Aim at an enemy in a stable posture. Increases Missile property attacks.
+SKILL_20160224_006215	Aim at an enemy in a stable posture. Increases the damage of Missile property attacks.
 SKILL_20160224_006216	Attack: +#{CaptionRatio}#%{nl}Additional Attribute Damage: #{CaptionRatio2}#{nl}Duration: 15 seconds
 SKILL_20160224_006217	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Bomb Duration: 2 seconds
-SKILL_20160224_006218	You can attack while moving quickly. Increases attack speed and deals additional damage when using basic attacks.
+SKILL_20160224_006218	Allows you to attack while moving quickly. Increases attack speed and deals additional damage when using basic attacks.
 SKILL_20160224_006219	{#DD5500}{ol}[Physical]{/}{/}{nl}Your companion holds on to a target. Decreases a target's physical defense and attacks against the target will have an increased critical chance.
 SKILL_20160224_006220	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Physical Defense: -#{CaptionRatio}#{nl}Additional Critical Chance: 30%{nl}Duration: #{CaptionTime}# seconds
 SKILL_20160224_006221	{#DD5500}{ol}[Physical]{/}{/}{nl}Order your companion to drop a Flying-type monster to the ground, afflicting [Bleeding].
 SKILL_20160224_006222	{#DD5500}{ol}[Physical]{/}{/}{nl}Order your companion to charge and attack an enemy.
-SKILL_20160224_006223	{#DD5500}{ol}[Physical]{/}{/}{nl}Order your companion to drag a target towards the player.
+SKILL_20160224_006223	{#DD5500}{ol}[Physical]{/}{/}{nl}Order your companion to drag a target towards you.
 SKILL_20160224_006224	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Maximum Duration: #{CaptionTime}# seconds
 SKILL_20160224_006225	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Duration: 6 seconds{nl}Consumes #{SpendPoison}# Poison Pot Poison
 SKILL_20160224_006226	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Attacked Targets {#339999}{ol}[Fear]{/}{/}+{#339999}{ol}[Confusion]{/}{/}{nl}Maximum Nearby #{CaptionRatio}#{#339999}{ol}[Fear]{/}{/}{nl}Duration: #{CaptionTime}# seconds
 SKILL_20160224_006227	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Duration: 10 seconds{nl}Attack Count: #{CaptionRatio2}#{nl}Consumes 1 Magic Arrow
 SKILL_20160224_006228	Attack: 200% + #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Duration: 10 seconds{nl}Consumes 1 Divine Machine Arrow
-SKILL_20160224_006229	Move while hanging on a hawk. You will not take damage from close ranged attacks and you can damage enemies with a basic attack while hanging.
+SKILL_20160224_006229	Move while hanging on your hawk. Prevents damage from close ranged attacks and you can damage enemies with basic attacks while hanging.
```

Comment:
this was changed so you can use skills while hanging off your hawk right? not just basic attacks